src/drivers/ulanding/ulanding.cpp

```diff
@@ -153,26 +204,20 @@ Radar::init()
 
 	do { /* create a scope to handle exit conditions using break */
 
-		/* do regular cdev init */
+#if defined(__PX4_POSIX)
+		/* do regular vdev init */
+		ret = VDev::init();
+#else
 		ret = CDev::init();
-
+#endif
 		if (ret != OK) {
-			PX4_WARN("cdev init failed");
-			break;
-		}
-
-		int fd = px4_open(RANGE_FINDER0_DEVICE_PATH, 0);
-
-		if (fd < 0) {
-			PX4_WARN("failed to open range finder device");
-			ret = 1;
+			PX4_WARN("vdev init failed");
 			break;
 		}
 
-		px4_close(fd);
-
 		/* open fd */
-		fd = px4_open(_port, O_RDWR | O_NOCTTY);
```

Comment:
`px4_` suffixed functions don't work out for you?

Reply:
They do not.